ml-agents-envs/mlagents/envs/brain.py

```diff
@@ -174,17 +174,19 @@ def from_agent_proto(agent_info_list, brain_params):
         return brain_info
 
 
-def safe_concat_lists(l1: Optional[List], l2: Optional[List]):
-    if l1 is None and l2 is None:
-        return None
-    if l1 is None and l2 is not None:
-        return l2.copy()
-    if l1 is not None and l2 is None:
-        return l1.copy()
+def safe_concat_lists(l1: Optional[List], l2: Optional[List]) -> Optional[List]:
```

Comment:
mypy was getting confused about the null-ness of the inputs. Personally I think this is a little nicer anyway.